job_spec/INSAR_ISCE.yml

```diff
@@ -84,6 +84,13 @@ INSAR_ISCE:
           - GMAO
           - HRES
           - HRRR
+    min_frame_coverage:
```

Comment:
And let's move the new parameter to right under the `frame_id` param, to match the container CLI.